Server/Plugins/APIDump/main_APIDump.lua

```diff
@@ -617,6 +620,14 @@ local function ReadDescriptions(a_API, a_Desc)
 					local NumInGroup = 0;
 					for _, incl in ipairs(group.Include or {}) do
 						for _, cons in ipairs(cls.Constants) do
+							if cons.Name:match(incl) then
+								if not cons.Name:match("^HOOK_") and not cons.Name:match("^cr") and not cons.Name:match("^ps")
+								and not cons.Name:match("^pk") and not cons.Name:match("^ba") and not cons.Name:match("^ms")
+								and not cons.Name:match("^bs") then
+									table.insert(GlobalsLuacheck, cons.Name)
```

Comment:
I don't get it, you exclude a lot of constants and then dump the rest without any namespace, is that how Luacheck needs to have them? I'd have expected Luacheck to either ignore classes' constants altogether, or use proper namespacing for them.